integration/spark/src/main/scala/org/apache/spark/util/AlterTableUtil.scala

```diff
@@ -652,6 +652,16 @@ object AlterTableUtil {
         throw new MalformedCarbonCommandException(
           s"Table property ${ CarbonCommonConstants.RANGE_COLUMN }: ${ rangeColumnProp }" +
           s" is not exists in the table")
+      }
+      val dataType = rangeColumn.getDataType.getName;
+      if (DataTypes.BINARY.getName.equalsIgnoreCase(dataType) ||
```

Comment:
This check is same as CarbonParserUtil range column check. can extract to method and reuse

Reply:
Done